catalogue_of_life/README.md

```diff
@@ -8,9 +8,8 @@ which enables us to leverage the rule-based inference capabilities of TypeDB and
 
 ## Prerequisites
 
-* System requirements: at least 16 GB of _free_ RAM (this is due to the size of the dataset)
+* System requirements: at least 7 GB of _free_ RAM
```

Comment:
It might be worth deleting this line altogether now.

Reply:
If this is an actual system requirement, then I would suggest retaining it. We don't want the scenario where a user runs out of memory and our README doesn't tell them why. However, it is unusual to specify free RAM as a requirement (as opposed to total RAM) since, generally speaking, if a program requests more RAM than is available, then memory from other inactive applications will be deallocated and given to the hungry program.

Could we replace "7GB of free RAM" with "8GB of RAM" or "12GB of RAM"?

Reply:
12 GB sounds reasonable.